bclientapi/bendoapi.go

```diff
@@ -23,7 +23,11 @@ var (
 func (ia *itemAttributes) GetItemInfo() (*jason.Object, error) {
 	var path = "http://" + ia.bendoServer + "/item/" + ia.item
 
-	r, err := http.Get(path)
+	req, _ := http.NewRequest("GET", path, nil)
+        if ia.token != "" { req.Header.Add("X-Api-Key", ia.token) }
```

Comment:
not sure how i feel having all this on one line. I guess it is okay if gofmt left it like this.